src/components/Calculator.js

```diff
@@ -2,6 +2,7 @@ import { useState } from 'react';
 import Button from './Button';
 import Body from './Body';
 import calculate from '../logic/calculate';
+import Quate from './Quot';
```

Comment:
- [ ] We should import the Quote component in the main component, that is the App component. Kindly fix the import.